config/rbac/operator_role.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  creationTimestamp: null
+  name: external-dns-operator
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+      - serviceaccounts
```

Comment:
Is this actually needed?  Granted, it isn't a significant security concern if the operator can manage serviceaccounts in its own namespace, but it isn't obvious to me why this is needed.

Reply:
By "this", I mean delete/get/list/update/watch access to serviceaccounts.